website/docs/usage/data-api.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 9
+---
+
+# Data API
+
+The FerretDB Data API is an open-source alternative to the MongoDB Atlas Data API.
+It lets you perform MongoDB-compatible operations via HTTP requests, without needing a MongoDB driver.
+The Data API is integrated directly into FerretDB – it's not a standalone service.
+
+## Enable the Data API
+
+To access the FerretDB Data API, set the [environment variable or flag](../configuration/flags.md) (`FERRETDB_LISTEN_DATA_API_ADDR`/`--listen-data-api-addr`) to the desired address and port when starting FerretDB.
+
+Suppose you want to run the Data API on port `8080` and your FerretDB instance is running locally on `localhost:27017`, set the environment variable or flag as `FERRETDB_LISTEN_DATA_API_ADDR=:8080` or `--listen-data-api-addr=:8080`.
+
+The Data API will be accessible at `http://localhost:8080`.
+Make sure to provide your authentication credential in the request headers or as part of the URL if authentication is enabled.
+
+## Using the Data API
+
+The Data API supports standard MongoDB operations like `insert`, `find`, `update`, and `delete`.
+It follows the [Data API OpenAPI 3.0 specification defined here](https://github.com/FerretDB/FerretDB/blob/refs/tags/v2.2.0/internal/dataapi/api/openapi.json).
+
+### Insert a document
+
+To insert a single document, use the `/action/insertOne` endpoint.
+
+```sh
+curl -X POST http://localhost:8080/action/insertOne \
+  -H "Content-Type: application/json" \
+  -u <username>:<password> \
+  -d '{
+        "database": "db",
+        "collection": "books",
+        "document": {
+          "_id": "pride_prejudice_1813",
```

Comment:
With CTS shouldn't this already exist and cause duplication error? Could we add non-existent book?

https://github.com/FerretDB/FerretDB/blob/2396a6caf1dff96872c5a41811cd443273de2628/website/docs/guides/fixtures/books.js#L3

But first, @noisersup please review this PR. Am I mistaken?

Reply:
It's the same example but we didn't use CTS here.